include/zim/file.h

```diff
@@ -61,7 +61,6 @@ namespace zim
       offset_type getClusterOffset(cluster_index_type idx) const;
 
       Blob getBlob(cluster_index_type clusterIdx, blob_index_type blobIdx) const;
-      offset_type getOffset(cluster_index_type clusterIdx, blob_index_type blobIdx) const;
```

Comment:
While I agree with the change, the whole File API will be dropped soon.
I don't think we need to introduce a API break just for few months (weeks).

And we don't know if there are other users than openzim and kiwix.

Reply:
I have dropped that commit now